app/src/main/java/com/klinker/android/twitter/activities/compose/Compose.java

```diff
@@ -1142,43 +1159,43 @@ protected Boolean doInBackground(String... args) {
                 Twitter twitter = Utils.getTwitter(getApplicationContext(), settings);
                 Twitter twitter2 = Utils.getSecondTwitter(getApplicationContext());
 
-                if (remaining < 0 && !pwiccer && !multiTweets) {
+                if (remaining < 0 && !pwiccer && !multiTweet) {
                     // twitlonger goes here
                     boolean isDone = false;
                     if (useAccOne) {
-                        isDone = tweetUsingTweetLonger(twitter);
+                        isDone = tweetUsingTwitLonger(twitter);
                     }
 
                     if (useAccTwo) {
-                        isDone = tweetUsingTweetLonger(twitter2);
+                        isDone = tweetUsingTwitLonger(twitter2);
                     }
 
                     return isDone;
-                } else if (multiTweets && remaining < 0) {
-                    /* We need to schedule this activity because we are hitting the
-                    *  rate limit quite easily. We will hardcode to schedule to tweeet every
-                    *  minute */
-                    List<String> multiTweets = getMultipeTweets(status);
-                    int noOfTweets = multiTweets.size();
+                } else if (multiTweet && remaining < 0) {
+                    Pair<String, List<String>> multiTweets = getMultipeTweets(status);
+                    int noOfTweets = multiTweets.second.size();
                     int tweetNo = 1;
                     for (int i = 0; i < noOfTweets; i++) {
-                        status = multiTweets.get(i) + "(" + tweetNo + "/" + noOfTweets + ")";
+                        status = multiTweets.first.length()!=0?multiTweets.first:"";
+                        status += multiTweets.second.get(i) + "(" + tweetNo + "/" + noOfTweets + ")";
+                        replyText = status.replace("/status/", "");
```

Comment:
The function tweetWithoutImages() checks for: replyText != null && !replyText.contains("/status/"), if that is the case, it will turn on autoPopulateMetadata.

Basically a way to setAutoPopulateReplyMetadata without passing in any additional parameter.